modules/cloudevent-broker/ingress.tf

```diff
@@ -47,6 +47,7 @@ module "this" {
       resources = {
         limits = var.limits
       }
+      regional-cpu-idle = var.cpu_idle
```

Comment:
what is the default value for this? i think the var.cpu_idel should match the default value